field_audit/models.py

```diff
@@ -0,0 +1,259 @@
+from datetime import datetime
+
+from django.conf import settings
+from django.contrib.postgres.indexes import GinIndex
+from django.db import models
+
+from .utils import class_import_helper, get_fqcn
+
+USER_TYPE_TTY = "SystemTtyOwner",
+USER_TYPE_PROCESS = "SystemProcessOwner"
+USER_TYPE_REQUEST = "RequestUser"
+
+
+def check_engine_sqlite(engine=None):
```

Comment:
where is `sqlite` used? It seems like tests run on PostgreSQL. If this is just for general compatibility it seems like a fair amount of added complexity for an unneeded use case (so far).

Also if we keep sqlite you'll want to add a note about JSON support: https://code.djangoproject.com/wiki/JSON1Extension

Reply:
Developing this library doesn't require PostgreSQL, and running tests locally uses SQLite. The GitHub Workflows tests use PostreSQL (because support for that is very important), but local tests don't need it.

Reply:
> Also if we keep sqlite you'll want to add a note about JSON support: https://code.djangoproject.com/wiki/JSON1Extension

Hmm, I haven't had any trouble with SQLite (running tests). Perhaps that extension is enabled on my system and I don't know it? I'll look further into this and add a note if it turns out SQLite isn't supported as-is.

Reply:
Addressed in a358992.